exercise3.R

```diff
@@ -0,0 +1,30 @@
+#Install tidyverse packages for this activity
+
+install.packages("tidyverse")
+install.packages("ggthemes")
+library(tidyverse)
+library(ggthemes)
+
+#Load malaria data Rstudio
+
+malaria_data <- read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/malaria.dat",header=TRUE)
+
+#View malaria data
+
+malaria_data
+
+#Assign time and cases as per the data
+
+time <- malaria_data$Time
+
+cases <- malaria_data$Cases
+
+rain <- malaria_data$Rain
+
+#plot malaria cases over time and labels
```

Comment:
Hi Bok, well done for this work.

I think here you are trying to answer Task 2 which was to plot the malaria cases over time. An easier way to do this would be to use the plot function. So once you have loaded the data set you can use the code:

plot(Cases, type = "l") 

and then to add in the time variable:

plot(as.factor(Time), Cases, type = "l")

When you run this code the graph will likely still not look right because it is treating r as a factor variable and not a character variable. we can convert it using the ts function:

cases.ts <- ts(Cases, start = c(1997, 7), frequency = 12)

Hope this helps!

Reply:
@EslamElbasheer, great review and great suggestions! Well done!

I agree with the comments. @Bokchol, please update the code in your branch as Eslam suggested. What she is showing is the base R approach to working with time series data. No use of `ggplot2` or `tidyverse` at all.

After implementing that and seeing what a time series trend plot would look like with base R, we can merge that in but I will challenge you also to figure out how to show the same plot but using `ggplot2`. If you can do that, that will tell me that you've understood how to do plots with `ggplot2`,